services/thermal_mgr/thermal_mgr.c

```diff
@@ -42,19 +42,37 @@ void initThermalSystemManager(lm75bd_config_t *config) {
 }
 
 error_code_t thermalMgrSendEvent(thermal_mgr_event_t *event) {
-  /* Send an event to the thermal manager queue */
-
+  xQueueSend(thermalMgrQueueHandle, event, portMAX_DELAY);
```

Comment:
Lots of things need to be changed here
- always check non-void functions for a return code
- check that the event and thermalMgrQueueHandle aren't null, returning the appropriate error codes
- the delay should be 0 and not portMAX_DELAY as that will block when called in isr if the queue is full which is not what we want